src/service.ts

```diff
@@ -552,6 +552,16 @@ export class Service {
     output.setData(s);
   }
 
+  static openBinaryExplorer(file: File) {
+    const childWindow = window.open("./lib/code-explorer.html", "", "toolbar=no,ocation=no,directories=no,status=no,menubar=no,location=no,scrollbars=yes,resizable=yes,width=1024,height=568");
```

Comment:
Not sure if we want to open a new window. I think, similar to #67, we want to open wasm as some preview. @mbebenita may have different idea for wasm file?

Reply:
Ah, opening as a tab would probably be better, but i think we need to first land the MD preview work, and make this fit into that. I think we should land this and live with a popup for a while, then integrate into a tab later.